extensions/sql-database-projects/src/models/connections/connectionService.ts

```diff
@@ -42,7 +42,7 @@ export class ConnectionService {
 				saveProfile: false,
 				id: '',
 				connectionName: profile.profileName,
-				options: [],
+				options: { 'encrypt': 'false' },
```

Comment:
Why is encrypt a string? That's very unintuitive - we should at least support both string and boolean if we're going to support string at all.

Reply:
It's a string as the values are now 'Mandatory', 'Optional', 'Strict', however we can keep the same support for backwards compat, adding it here: https://github.com/microsoft/sqltoolsservice/pull/1737